src/main/java/Luke.java

```diff
@@ -0,0 +1,205 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Luke {
```

Comment:
Maybe enhance the class structure by decomposing the functionality into separate classes such as InputValidator for validating user inputs, Outputs for managing output operations, and InputInterpreter for processing user commands, thereby promoting a clearer separation of concerns.